<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorOptions</class>
 <widget class="QDialog" name="SensorOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QPushButton" name="clear_bluetooth_devices_btn">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="baudrate">
     <property name="text">
      <string>115200</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="scan_bluetooth_devices_btn">
     <property name="text">
      <string>Scan for Device</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QListWidget" name="list_bluetooth_ports"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Baudrate:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDialogButtonBox" name="sensorOptionButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sensorOptionButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SensorOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensorOptionButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SensorOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
